### PR TITLE
Label csv source feature

### PR DESCRIPTION
**Pull request by sudharsana-kjl**
This PR enables setting label and corresponding LabelColumn in CSVSource.

### CODE REVIEW

**Comment by sudharsana-kjl** [comment]
@pdxjohnny  This PR's branch is based on another branch used in #102. Once that PR is merged, those changes will be gone.

**Comment by pdxjohnny** [comment]
I'm working on fixing the merge test. I'll fix that for JSONSource, modify things here, and merge it all soon.

**Comment by pdxjohnny** [comment]
Let's add a property `write_back_label` to `OpenCSVFile` class which is a `bool` that determines if we are going to write the `label_column` out when doing `dump_fd`. It should be set to true if the label column was present when the source was loaded.

File: dffml/source/csv.py [hunk 1]
```diff
@@ -68,8 +69,11 @@ class CSVSource(FileSource, MemorySource):
                     data=[], active=1, lock=asyncio.Lock()
                 )
                 if fd is not None:
+                    dict_reader = csv.DictReader(fd, dialect="strip")
+                    if self.config.label_column in dict_reader.fieldnames:
+                        self.OPEN_CSV_FILES[self.config.filename].write_back_label = 1
```
Comment (pdxjohnny) [inline dffml/source/csv.py]: Nice!

**Pull request merged**